gsadjust/pyqt_models.py

```diff
@@ -556,6 +572,27 @@ def unique_meters(self):
             meters.append(station.meter[0])  # Get the first entry; Assume meter number can't change at a station
         return list(set(meters))
 
+    @property
+    def loops(self):
+        return [self.child(i) for i in range(self.rowCount())]
+    
+    # @property
```

Comment:
I was a bit confused about this actually, as we seem to have a `.deltas` property on this object, and set it when loading (if this prop is present it will error out) but when writing we build the deltas list as shown below. I couldn't see where the .deltas was actually used, perhaps it's better to do as below + add a getter/setter pair.